examples/utils/request.py

```diff
@@ -0,0 +1,57 @@
+import requests
```

Comment:
As we discussed, let's
1) move this to `utils/utils.py`
2) use something similar that I sent over in Slack

Reply:
done